src/custom/hooks/web3.ts

```diff
@@ -1,22 +1,35 @@
-import { useWeb3React as useWeb3ReactCore } from '@web3-react/core'
+import { IS_IN_IFRAME, NetworkContextName } from 'constants/misc'
+import { Web3Provider } from '@ethersproject/providers'
+import { useWeb3React } from '@web3-react/core'
 import { AbstractConnector } from '@web3-react/abstract-connector'
 import { useEffect, useState, useCallback } from 'react'
-import { isMobile } from 'react-device-detect'
-import { injected, walletconnect, getProviderType, WalletProvider, fortmatic, walletlink } from 'connectors'
+
+import { injected, gnosisSafe, walletconnect, getProviderType, WalletProvider, fortmatic, walletlink } from 'connectors'
 import { STORAGE_KEY_LAST_PROVIDER } from 'constants/index'
+import { isMobile } from 'utils/userAgent'
 
 // exports from the original file
-export { useActiveWeb3React, useInactiveListener } from '@src/hooks/web3'
+export { useInactiveListener } from '@src/hooks/web3'
+
+export function useActiveWeb3React() {
```

Comment:
Looks like this function stayed the same as the original one, is there a reason why we re-declared it here instead of just exporting it from the original like we did before?